frontend/src/Routes.tsx

```diff
@@ -15,6 +16,7 @@ export const Routes: React.FC = () => (
     }
   >
     <Switch>
+      <Route exact path="/testK8s" component={TestK8s} />
```

Comment:
Good for first round, but we should consider creating dynamic plugin to show the full path.